src/store-api/src/region_request.rs

```diff
@@ -32,6 +34,85 @@ pub enum RegionRequest {
     Compact(RegionCompactRequest),
 }
 
+impl RegionRequest {
+    /// Convert [Body](region_request::Body) to a group of [RegionRequest] with region id.
+    /// Inserts/Deletes request might become multiple requests. Others are one-to-one.
+    // TODO: implement alter request
+    #[allow(unreachable_code)]
+    pub fn from_request_body(
```

Comment:
This should be try_from
```suggestion
    pub fn try_from_request_body(
```